[BUG, RR2] Makefile is broken

Hi!

> Thanks for `RR v2`.

I cannot understand if this is a **bug** or if something is not working for me.

I tried this code:

```sh
$ make build
```
I expected to see this happen: `build successfull`

Instead, this happened: 
```sh
CGO_ENABLED=0 go build -trimpath -ldflags "-s" -o ./rr ./cmd/main.go
stat /proj/spiral/roadrunner/cmd/main.go: directory not found
Makefile:15: recipe for target 'build' failed
make: *** [build] Error 1
```

I do not see the `cmd` directory

thanx a lot.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,19 @@
 CHANGELOG
 =========
 
+v2.3.0 (01.06.2021)
+-------------------
+
+## 👀 New:
+
+- ✏️
+
+## 🩹 Fixes:
+
+- 🐛 Fix: remove `build` and other old targets from the Makefile.
+
+---
+
 v2.2.0 (11.05.2021)
 -------------------
 

--- a/Makefile
+++ b/Makefile
@@ -4,25 +4,6 @@
 
 SHELL = /bin/sh
 
-.DEFAULT_GOAL := build
-
-# This will output the help for each task. thanks to https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
-help: ## Show this help
-	@printf "\033[33m%s:\033[0m\n" 'Available commands'
-	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z0-9_-]+:.*?## / {printf "  \033[32m%-14s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
-
-build: ## Build RR binary file for local os/arch
-	CGO_ENABLED=0 go build -trimpath -ldflags "-s" -o ./rr ./cmd/main.go
-
-clean: ## Make some clean
-	rm ./rr
-
-install: build ## Build and install RR locally
-	cp rr /usr/local/bin/rr
-
-uninstall: ## Uninstall locally installed RR
-	rm -f /usr/local/bin/rr
-
 test_coverage:
 	docker-compose -f tests/docker-compose.yaml up -d --remove-orphans
 	rm -rf coverage

